dojo/endpoint/views.py

```diff
@@ -228,7 +228,7 @@ def delete_endpoint(request, eid):
                                      messages.SUCCESS,
                                      'Endpoint and relationships removed.',
                                      extra_tags='alert-success')
-                return HttpResponseRedirect(reverse('view_endpoint', args=(product.id,)))
+                return HttpResponseRedirect(reverse('view_product', args=(product.id,)))
```

Comment:
I like this change. Only qualm I have is if a user is deleting multiple endpoints. It would be super annoying to navigate back to All Endpoints for every one to be deleted. Maybe there is a way to redirect to product only if there are no endpoints left? Thoughts from other reviewers?

Reply:
There was an error there.
after deleteing any endpoint the program crashes.. because it's trying to redirect back to the same endpoint that was just deleted

Reply:
and it returns an Id not found error.

Reply:
Got it. Good find.